keyboards/sirius/unigo66/main.c

```diff
@@ -1,31 +1,31 @@
-#include <avr/io.h>
 #include <avr/wdt.h>
 #include <avr/power.h>
-#include <util/delay.h>
 
 // LUFA
 #include "lufa.h"
 
 #include "sendchar.h"
 #include "debug.h"
 #include "keyboard.h"
+#include "gpio.h"
+#include "wait.h"
 #include "led.h"
 
 /* LED ping configuration */
 #define TMK_LED
 //#define LEONARDO_LED
 #if defined(TMK_LED)
 // For TMK converter and Teensy
-#define LED_TX_INIT    (DDRD  |=  (1<<6))
-#define LED_TX_ON      (PORTD |=  (1<<6))
-#define LED_TX_OFF     (PORTD &= ~(1<<6))
-#define LED_TX_TOGGLE  (PORTD ^=  (1<<6))
+#define LED_TX_INIT    gpio_set_pin_output(D6)
+#define LED_TX_ON      gpio_write_pin_high(D6)
+#define LED_TX_OFF     gpio_write_pin_low(D6)
+#define LED_TX_TOGGLE  gpio_toggle_pin(D6)
 #elif defined(LEONARDO_LED)
 // For Leonardo(TX LED)
-#define LED_TX_INIT    (DDRD  |=  (1<<5))
-#define LED_TX_ON      (PORTD &= ~(1<<5))
-#define LED_TX_OFF     (PORTD |=  (1<<5))
-#define LED_TX_TOGGLE  (PORTD ^=  (1<<5))
+#define LED_TX_INIT    gpio_set_pin_output(D5)
+#define LED_TX_ON      gpio_write_pin_high(D5)
+#define LED_TX_OFF     gpio_write_pin_low(D5)
```

Comment:
This is switched from the previous implementation, which was D5 low for on, and high for off.

Error in the PR, or error in the existing (previous) code?